From b9debbf540329c1dee28d707e47e0694244bf9db Mon Sep 17 00:00:00 2001
From: changfengpro <2728019272@qq.com>
Date: Fri, 17 Apr 2026 11:32:03 +0800
Subject: [PATCH 03/13] fix: HAL_INTMUX_Init

---
 project/rk3506-mcu/src/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/project/rk3506-mcu/src/main.c b/project/rk3506-mcu/src/main.c
index 81737a4..83957dd 100644
--- a/project/rk3506-mcu/src/main.c
+++ b/project/rk3506-mcu/src/main.c
@@ -112,6 +112,7 @@ int main(void)
 {
     HAL_Init();
     BSP_Init();
+    HAL_INTMUX_Init();
     BSP_UART_Init();
 
 #ifdef RPMSG_TEST
-- 
2.43.0

